### ISSUE TITLE
[Legitimate Site Unblocked] cryptounion-trades.co 

### ISSUE THREAD

**Issue by Cryptolover321** [MetaMask/eth-phishing-detect#12555]
**Domain:**Cryptounion-trades.co 

**Details:** *Some details about the domain (optional)*

- [x] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by Cryptolover321** [comment]
This site was not metamask related why was it flagged

**Comment by Cryptolover321** [comment]
What documents you need, this is not a metamask related nor a fishing
website, so why was it flagged

**Comment by AlexHerman1** [comment]
1. What services does your site offer?
2. How long has your business been in service?
3. What social media sites do you have a presence on?

**Comment by Dayplayyyy** [comment]
It's none of metamask`s business what anyone does with their site as long as it's not metamask related y'all are just doing rubbish that's how you flagged my legit site also

**Comment by Cryptolover321** [comment]
This was neither metamask nor a phising site please release this website

**Comment by AlexHerman1** [comment]
Hi there,
This is not blocked on our list, I will reach out to third party that blocked. In the meantime, if you could open a new PR with the information I've asked for, that would be helpful.

**Comment by Cryptolover321** [comment]
Please kindly review my website

**Comment by Cryptolover321** [comment]
The website is not metamask related
